{{ cookiecutter.repo_name }}/docs/structure/README.md

```diff
@@ -1,149 +1,124 @@
+sourcing environment variables, for example in conjunction with PyCharm's EnvFile plugin.
+{% if cookiecutter.create_secrets_file == "Yes" %}
+To ensure this `sed` command works correctly, make sure any file paths listed in this file, and the
+[`.secrets`](#secrets) are absolute file paths, or relative file paths that do not use other environment variables.
+For example:
+{% else %}
+To ensure this `sed` command works correctly, make sure any file paths listed in this file are absolute file paths, or
+relative file paths that do not use other environment variables. For example:
+{% endif %}
+```shell
+export DIR_DATA=$(pwd)/data  # fine
+export DIR_DATA_EXTERNAL=$(pwd)/data/external  # fine
+export DIR_DATA_EXTERNAL=./data/external  # fine
+export DIR_DATA_EXTERNAL=$DIR_DATA/external  # will break the `sed` command!
+```
 
 ### `.flake8`
 
-A configuration file for the [`flake8`](https://gitlab.com/pycqa/flake8) Python package that provides linting. This
-file is based on the
-[common configuration](https://gds-way.cloudapps.digital/manuals/programming-languages/python/python.html#common-configuration)
-described on [The GDS Way](https://gds-way.cloudapps.digital).
+A configuration file for the [`flake8`][flake8] Python package that provides linting. This file is based on the
+[common configuration][gds-way-flake8] described on [The GDS Way][gds-way].
 
 ### `.gitignore`
 
 A `.gitignore` file to ignore certain files and folders from this Git repository. See the
-[GitHub Help pages](https://help.github.com/en/github/using-git/ignoring-files) for further information.
+[contributor guide][docs-updating-gitignore] for further information about modifying this file.
 
 ### `.pre-commit-config.yaml`
 
-A pre-commit hook configuration file. See the `pre-commit` package [documentation](https://pre-commit.com/) for further
-details.
+A pre-commit hook configuration file. See the [contributor guide][docs-pre-commit-hooks] for further details.
 {% if cookiecutter.create_secrets_file == "Yes" %}
 ### `.secrets`
 
 A file to store all secrets and credentials as environment variables. This is read-in by [`.envrc`](#envrc) using the
-[`direnv`](https://direnv.net/) shell extension, but is **not** tracked by Git.
+[`direnv`][direnv] shell extension, but is **not** tracked by Git.
 {% endif %}
 ### `.secrets.baseline`
 
-Optional baseline file for the [`detect-secrets`](https://github.com/Yelp/detect-secrets) package; this package detects
-secrets, and, in conjunction with `pre-commit`, prevents them from being committed to the repository. The baseline file
-flags secret-like data that the user deliberately wishes to commit the to repository.
+Baseline file for the [`detect-secrets`][detect-secrets] package; this package detects secrets, and, in conjunction
+with `pre-commit`, prevents them from being committed to the repository. The baseline file flags secret-like data that
+the user deliberately wishes to commit the to repository.
```

Comment:
Shouldn't the `secret-baseline` only be displayed IFF `{% if cookiecutter.create_secrets_file == "Yes" %}`?

Reply:
Ah no, `.secrets.baseline` deals with the [`detect-secrets`](https://github.com/Yelp/detect-secrets) pre-commit hook. So this deals with preventing secrets ending up in the repo, whereas the `.secrets` file is for storing secrets locally only, and not committing them.

Reply:
I thought the same as @exfalsoquodlibet! 😅

Reply:
Despite the [`detect-secrets`] documentation, I'm pretty certain we could rename the `.secrets.baseline` file to something more else (and update the documentation accordingly) -- do you think this will resolve the ambiguity @exfalsoquodlibet and @avisionh?

Reply:
Part of me thinks we shouldn't since .`secrets.baseline` is related to `.secrets`, right? Thus, if we change the name to something different, then it would become less intuitive that the two files are linked. Maybe we just need to clarify that `create_secrets_file` creates a `.secrets` file and a `.secrets.baseline` file is different but related.

Reply:
I'm going to mark this as resolved - we'll be making `.secrets` creation mandatory in a future PR, so that should resolve any confusion around what `create_secrets_file` does (as it'll be removed!).